### PR TITLE
Update .cctemplate

### PR DESCRIPTION
**Pull request by leoervik**
IntelliJ is unable to resolve .cc templates with the missing paths

### CODE REVIEW

File: .cctemplate [hunk 1]
```diff
@@ -69,16 +69,6 @@
             "name": "☸️ k8s-nodejs-guestbook",
             "description": "k8s NodeJS guestbook"
         },
-        {
-            "path": "platforms/gke-wa-shop",
```
Comment (redvg) [inline .cctemplate:73]: what would be the reason to delete these 2 templates?

**Review by redvg — CHANGES_REQUESTED** [review]
nice catch with 2 typos. but not sure what is the reason to delete 2 templates

File: .cctemplate (hunk 1 above)
Comment (leoervik) [reply .cctemplate:73]: they don't exist

Comment (leoervik) [reply .cctemplate:73]: @redvg

**Pull request closed without merging**